docs/docs/endpoints-test/redis.md

```diff
@@ -50,6 +50,117 @@ The `Health` function returns a JSON-like map structure with various keys repres
 }
 ```
 
+#### NOTE: The `Sample Output` is dynamic and unstructured since it depends on the raw map. To make it structurable, it must implement `JSON serialization/deserialization` or `Other serialization/deserialization` (e.g, `XML serialization/deserialization`) to bind it. For example:
```

Comment:
Brake after 'NOTE:' into a new line, as it is interfering with the table of contents

![image](https://github.com/Melkeydev/go-blueprint/assets/116409846/52ae05f2-2be3-40e6-9a02-3b77a77677e6)

Reply:
@Ujstor cek latest commit

Reply:
@H0llyW00dzZ  Looks awkward, I think this is better. Sorry, my mistake

#### Note: serialization/deserialization

![image](https://github.com/Melkeydev/go-blueprint/assets/116409846/ecdc971d-878f-45fc-8012-e3e6040ac7b6)

Reply:
@Ujstor cek I revert it

Reply:
@H0llyW00dzZ 

No, now is the same as before....I am nit... put it this way:

```md
#### Serialization/deserialization

The `Sample Output` is dynamic and unstructured since it depends on the raw map. To make it structurable, it must implement `JSON serialization/deserialization` or `Other serialization/deserialization` (e.g, `XML serialization/deserialization`) to bind it. For example:
```

Reply:
@Ujstor cek